src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,6 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
we need some permission right? at minimum we need to be able to store the user ID , last event timestamp and session id. I am not sure about other device information permission like network carrier.

Reply:
the only permission needed currently is internet. save userID in preference do not need storage permission. 
and do you want to add permission to get network carrier ? if that permission should be in sdk. then we can also get imei automatically in sdk.

Reply:
then no need to add the permission. It is too sensitive to get imei.